arkh/chain.json

```diff
@@ -3,7 +3,7 @@
   "chain_name": "arkh",
   "status": "live",
   "network_type": "mainnet",
-  "website": "https://arkh.com/",
+  "website": "http://arkhadian.com/",
```

Comment:
Site doesn't work? It the domain reserved?

Reply:
nah, the site is currently down But the previous site has no relation to this chain